### PR TITLE
Order Allocation allocate to member

### PR DESCRIPTION
**Pull request by brettski**
Refactor 'me' path for orders
add eventId filter for orders all query
allocate to logic in. doesn't check for current tickets

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -7,6 +7,10 @@ import * as Sentry from '@sentry/node';
 import { v4 as uuidv4 } from 'uuid';
 
 import apolloGraphServer from './graphql';
+import orderAllocationEventEmitter from './events/orderAllocation';
+
+const orderAllocationEvents = orderAllocationEventEmitter();
+console.log('order allocation', orderAllocationEvents);
```
Comment (csell5) [inline src/index.js:13]: did you want to leave this in?

File: src/graphql/typeDefs/mutations/meOrderAllocation.graphql [hunk 2]
```diff
@@ -0,0 +1,4 @@
+type MeOrderAllocationMutation {
+  "assigns Order Allocation to a member"
+  allocateTo(email: EmailAddress!): AllocateToResult!
```
Comment (csell5) [inline src/graphql/typeDefs/mutations/meOrderAllocation.graphql:3]: no role required?

File: src/graphql/typeDefs/mutations/meOrder.graphql [hunk 3]
```diff
@@ -0,0 +1,4 @@
+type MeOrderMutation {
+  "Mutation actions on OrderAllocation"
+  orderAllocation(orderAllocationId: ID!): MeOrderAllocationMutation
```
Comment (csell5) [inline src/graphql/typeDefs/mutations/meOrder.graphql:3]: no role required?

File: src/graphql/typeDefs/mutations/orderAllocation.graphql [hunk 4]
```diff
@@ -0,0 +1,7 @@
+"Internal/Admin Order Allocation mutations"
+type OrderAllocationMutation {
+  "assigns Order Allocation to a member"
+  allocateTo(member: AllocateToInput!): Boolean!
```
Comment (csell5) [inline src/graphql/typeDefs/mutations/orderAllocation.graphql:4]: no role required?

File: src/graphql/typeDefs/mutations/orderAllocation.graphql [hunk 5]
```diff
@@ -0,0 +1,7 @@
...
+  "Marks Order Allocation as refunded and removes Allocated Member"
+  setRefunded: Boolean!
```
Comment (csell5) [inline src/graphql/typeDefs/mutations/orderAllocation.graphql:6]: no role required?

File: src/events/orderAllocation.js [hunk 6]
```diff
@@ -0,0 +1,116 @@
...
+        month: new Intl.DateTimeFormat('en-US', { month: 'long' }).format(
+          event.startDate,
+        ),
+      },
+    };
+
+    return sendTransactionalEmail({
+      mailTo: memberTo.email,
+      templateModel,
+      templateAlias: 'product-allocated-to-message',
```
Comment (csell5) [inline src/events/orderAllocation.js:92]: don't forget to set this in production

**Review by csell5 — CHANGES_REQUESTED** [review]

File: src/index.js (hunk 1 above)
Comment (brettski) [reply src/index.js:13]: oops. removed.

File: src/graphql/typeDefs/mutations/meOrder.graphql (hunk 3 above)
Comment (brettski) [reply src/graphql/typeDefs/mutations/meOrder.graphql:3]: fixed, missed the whole path

File: src/graphql/typeDefs/mutations/meOrderAllocation.graphql (hunk 2 above)
Comment (brettski) [reply src/graphql/typeDefs/mutations/meOrderAllocation.graphql:3]: fixed. missed the whole path

File: src/graphql/typeDefs/mutations/orderAllocation.graphql (hunk 4 above)
Comment (brettski) [reply src/graphql/typeDefs/mutations/orderAllocation.graphql:4]: Not really, handled at the parent, though added for consistency.

File: src/events/orderAllocation.js (hunk 6 above)
Comment (brettski) [reply src/events/orderAllocation.js:92]: There is no message to put up yet. I purposely didn't create a placeholder so we don't end up with a coonference

**Comment by brettski** [comment]
Corrections pushed in 6ec4ef7

**Review by csell5 — APPROVED** [review]

**Pull request merged**